src/scanners/css.js

```diff
@@ -31,25 +31,23 @@ export default class CSSScanner extends BaseScanner {
     }
 
     var file = cssCode.position.source;
-    var startLine = cssCode.position.start.line;
-    var startColumn = cssCode.position.start.column;
+    var cssInfo = Object.assign({}, this.options, {
```

Comment:
Maybe `opts` or `cssOptions` would be better as this is called options in the actual rules?